src/components/PokeDetails.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+export default class PokeDetails extends Component {
+    render() {
+        return (
+            <div>
+                 <img src={this.props.url_image} alt={this.props.name} />
+                 <div>
```

Comment:
It would be gold to change these divs to <p> or <h2>, etc for more semantic html and to avoid div soup